pkg/asset/agent/installconfig.go

```diff
@@ -110,6 +113,29 @@ func (a *OptionalInstallConfig) validateSupportedArchs(installConfig *types.Inst
 	var allErrs field.ErrorList
 
 	fieldPath := field.NewPath("ControlPlane", "Architecture")
+	releaseImage := &releaseimage.Image{}
+	asseterr := releaseImage.Generate(asset.Parents{})
+	if asseterr != nil {
+		allErrs = append(allErrs, field.InternalError(fieldPath, asseterr))
+	}
+	cmd := fmt.Sprintf(templateGetReleaseImageArch, releaseImage.PullSpec)
+	releaseArch, err := OCexecute(installConfig.PullSecret, cmd)
+	if err != nil {
+		logrus.Warnf(fmt.Sprintf("Unable to validate the release image architecture"))
+	} else {
+		releaseArch = strings.Trim(releaseArch, "'")
+		// Validate that the release image supports the install-config architectures.
+		switch releaseArch {
+		// Check the release image to see if it is multi, if so fail. We don't support multi release images at install time (yet).
+		case "multi":
+			allErrs = append(allErrs, field.Forbidden(fieldPath, fmt.Sprintf("multi architecture release image %s is not supported", releaseImage.PullSpec)))
+		// if the release image isn't multi, then its single arch, and it must match the cpu architecture.
+		case string(installConfig.ControlPlane.Architecture):
```

Comment:
@andfasano as of now.. 'multi' isn't supported for agent installer. If this release image is detected, then we fail. Enabling the use of multi will require additional work in openshift-install and assisted-service (something we've targeted for 4.15)